src/components/UTIL_PageMessages.component

```diff
@@ -15,7 +15,7 @@
         </style>
     </apex:outputText>
 
-    <div class="slds" id="page_messages">
```

Comment:
Checked one of the other pages using the sdls static resources that also uses this component. Errors look fine.